lib/lightning_web/live/workflow_live/github_sync_modal.ex

```diff
@@ -25,7 +25,7 @@ defmodule LightningWeb.WorkflowLive.GithubSyncModal do
   @impl true
   def render(assigns) do
     ~H"""
-    <div>
+    <form id={@id <> "-form"} phx-submit="save">
```

Comment:
Hey @midigofrank nice catch! Would you mind renaming the event for `save_gh`? I find it important because it is handled by the `WorkflowLive.Edit` which we know that needs to be refactored into smaller components.

Reply:
Hey @jyeshe , good idea, I actually thought about and decided just to leave it as it was under the `def handle_event("save", _, _)`  callback.

The logic is tightly coupled with the usual `save action` with the only difference being the call to github sync and the flash message, so I just decided to leave it coupled given the time. 

Let me see if I can refactor it out easily

Reply:
Hey @jyeshe , I have refactored it out into a different callback. Let me know your thoughts